client/app/bundles/course/survey/containers/ShowQuestion.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { PropTypes } from 'react';
+import { injectIntl, defineMessages } from 'react-intl';
+import { connect } from 'react-redux';
+import { showDeleteConfirmation } from '../actions';
+import { formatQuestionFormData } from '../utils';
+import QuestionCard from '../components/QuestionCard';
+import * as questionActions from '../actions/questions';
+
+const translations = defineMessages({
+  editQuestion: {
+    id: 'course.surveys.Question.editQuestion',
+    defaultMessage: 'Edit Question',
+  },
+  updateSuccess: {
+    id: 'course.surveys.Question.updateSuccess',
+    defaultMessage: 'Question updated.',
+  },
+  updateFailure: {
+    id: 'course.surveys.Question.updateFailure',
+    defaultMessage: 'Failed to update question.',
+  },
+  deleteQuestion: {
+    id: 'course.surveys.Question.deleteQuestion',
+    defaultMessage: 'Delete Question',
+  },
+  deleteSuccess: {
+    id: 'course.surveys.Question.deleteSuccess',
+    defaultMessage: 'Question deleted.',
+  },
+  deleteFailure: {
+    id: 'course.surveys.Question.deleteFailure',
+    defaultMessage: 'Failed to delete question.',
+  },
+});
+
+class ShowQuestion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.updateQuestionHandler = this.updateQuestionHandler.bind(this);
```

Comment:
I find that a shorthand for defining bind method is:
```
updateQuestionHandler = (data) => {

}
```

in the above case, `updateQuestionHandler` will autobind to the current object.